source/extensions/filters/http/jwt_authn/extractor.cc

```diff
@@ -165,8 +179,11 @@ std::vector<JwtLocationConstPtr> ExtractorImpl::extract(const Http::HeaderMap& h
         }
         value_str = value_str.substr(location_spec->value_prefix_.size());
       }
-      tokens.push_back(std::make_unique<const JwtHeaderLocation>(
-          std::string(value_str), location_spec->specified_issuers_, location_spec->header_));
+      JwtLocationConstPtr location = std::make_unique<const JwtHeaderLocation>(
```

Comment:
Ideally, not to check all headers or parameters.  only check these headers and parameters specified by the required providers

Reply:
done.